src/Symfony/Component/Console/Attribute/CommandAutoTag.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Attribute;
+
+use Symfony\Contracts\Service\Attribute\AutoTag;
+
+/**
+ * Tells when a class or an interface should autoconfigured as a command.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class CommandAutoTag extends AutoTag
+{
+    public const NAME = 'console.command';
+
+    public function __construct(string $name)
+    {
+        parent::__construct(tags: [[
```

Comment:
Based on Travis, this line produces a syntax error on PHP 7.2 🤔

Reply:
outdated CI, now rebased